backbone.js

```diff
@@ -1440,6 +1440,11 @@
       return path === this.root && !this.getSearch();
     },
 
+    // Decode unicode escapes without decoding `%25`.
```

Comment:
Also, any reason this has to be on the Router prototype instead of living as a standalone helper function?